kolibri/core/assets/src/views/PaginatedListContainer.vue

```diff
@@ -150,7 +150,7 @@
 
   .pagination-nav {
     margin-bottom: 8px;
-    text-align: right;
+
```

Comment:
I believe replacing `text-align: right` with `dir="auto"` will not have the intended behavior